bootstrap/node.go

```diff
@@ -32,6 +32,9 @@ func NewNode(
 	nodePrivateKey primitives.Ed25519PrivateKey,
 	federationNodes map[string]config.FederationNode,
 	blockSyncCommitTimeoutMillis uint32,
+	blockTransactionReceiptQueryStartGraceSec uint32,
```

Comment:
See #74. This function is taking too many parameters and it's imperative that we spend some time fixing it. Please take some time along with @IdoZilberberg and pay back this technical debt this week.

Reply:
correct
but maybe we can try and do a fuel tank test on it, i.e. how low can it go before the car stops, so how many can we accumulate here until no-one can add anything because its too big? :) (talking to ido)

Reply:
I don't object to merging this PR without handling the config, as long as we *commit* to fixing the config problem this week.